docset/windows/international/Copy-UserInternationalSettingsToSystem.md

```diff
@@ -0,0 +1,89 @@
+
+---
+ms.mktglfcycl: manage
+ms.sitesec: library
+ms.author: ertorres
+author: Erik Torres
+description: Use this topic to help manage Windows and Windows Server technologies with Windows PowerShell.
+external help file: Microsoft.InternationalSettings.Commands.dll-Help.xml
+keywords: powershell, cmdlet
+manager: aandrejs
+ms.date: 10/23/2020
+ms.prod: w10
+ms.technology: powershell-windows
+ms.topic: reference
+online version: 
+schema: 2.0.0
+title: Copy-UserInternationalSettingsToSystem
+ms.reviewer:
+ms.assetid: 
+---
+
+# Copy-UserInternationalSettingsToSystem
+
+## SYNOPSIS
+Copy the current user's international settings (Display language, Input language, Format/locale and Location/GeoID) to one or both of the following:
+    * Welcome screen and system accounts
+    * New user accounts
+
+## SYNTAX
+
+```
+Copy-UserInternationalSettingsToSystem [-WelcomeScreen]<Boolean> [-NewUser]<Boolean> [<CommonParameters>]
+```
+
+## DESCRIPTION
+The **Copy-UserInternationalSettingsToSystem** cmdlet gets the **international settings** from the current user.
+You can decide whether to copy this settings into the **Welcome screen and system accounts**, the **New user accounts** or both.
+
+This cmdlet needs two parameters: **-WelcomeScreen** and **-NewUser**.
+
+When any of the parameters is set to $True, this gets the current values from the registry for the current user and copies them into the selected places whose parameter was set to $True.
```

Comment:
```suggestion
When any of the parameters is set to $True, this gets the current values from the current user and copies them to the system settings for the selected options based on the parameters that were set to $True.
```

Reply:
lets not mention the registry, as the cmdlet doesn't actually read from the registry.
(it uses your new API).